src/QuestionInput/QuestionInput.js

```diff
@@ -7,20 +7,22 @@ const QuestionInput = (props) => {
   return (
     <div className={`trip-${name}-field`}>
         <h2 className='question-input-question'>{question}</h2>
-        {type === "text" && <input
+        {type === 'text' && <input
```

Comment:
Someone fixed the double quotation marks! Thank you! :)

Reply:
WOW!!! love it